packages/server/graphql/mutations/helpers/createNewMeetingPhases.ts

```diff
@@ -86,7 +87,7 @@ const createNewMeetingPhases = async (
   const facilitatorTeamMemberId = toTeamMemberId(teamId, facilitatorUserId)
   const asyncSideEffects = [] as Promise<any>[]
 
-  const {tier = 'starter'} = team ?? {}
+  const tier = team ? getFeatureTier(team) : 'starter'
```

Comment:
+1 We should not create meetings without a team, so really l84 should be changed to
```ts
    dataLoader.get('teams').loadNonNull(teamId)
```

Reply:
Fixed